### ISSUE TITLE
not working with AWS SSO?

### ISSUE THREAD

**Issue by allixsenos** [segmentio/chamber#303]
We recently moved from hardcoded IAM accounts to AWS SSO, using `aws sso login`.

Most tools don't seem to care (other than failing when there's no active and valid SSO session), but chamber doesn't like this new setup at all.

Running with aws-vault makes it work, but we were expecting the native way to work as well.

```ini
$ AWS_PROFILE=test-engineer chamber list airflow
Error: Failed to list store contents: NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
$ aws-vault exec test-engineer -- chamber list airflow
Key							Version		LastModified		User
AIRFLOW_CONN_AWS_DEFAULT				0		2020-11-10 15:00:53
AIRFLOW_CONN_DATADOG_DEFAULT				0		2020-11-10 15:00:55
```

our ~/.aws/config looks like this:
```ini
[profile test-engineer]
sso_role_name=Engineer
sso_account_id=AWSACCOUNTID
sso_start_url=https://COMPANYNAME.awsapps.com/start
sso_region=us-east-1
region=us-east-1
```

There's no ~/.aws/credentials file.

Please advise

**Comment by nickatsegment** [comment]
Should be fixed by #304. Will cut a release :)

**Comment by nickatsegment** [comment]
v2.10.0
